charts/telemetry-collector/values.yaml

```diff
@@ -0,0 +1,165 @@
+global: {}
+
+nameOverride: ""
+fullnameOverride: ""
+
+replicaCount: 1
+
+bootstrap:
+  create: true
+  name: ""
+  kafka:
+    enabled: true
+    hostname: drax-kafka
+    port: 0
+  nats:
+    enabled: true
+    hostname: drax-nats
```

Comment:
It should be empty, but we're not keeping it then...

Reply:
The whole bootstrap section is gone now